mappings-overrides.json

```diff
@@ -419,6 +419,11 @@
     "action": "execute",
     "type": "procedural"
   },
+  "orders.bypass-acquisition-units": {
+    "resource": "Orders Bypass-Acquisition-Units",
+    "action": "manage",
+    "type": "data"
```

Comment:
Does this permission protect some procedure?

Reply:
Yes. [example](https://github.com/folio-org/mod-orders/blob/master/src/main/java/org/folio/helper/PurchaseOrderHelper.java#L382)

Reply:
if yes why its has type as `data` and action `manage`.. we have a specific type for procedures

Reply:
Similar to `invoices.bypass-acquisition-units`. And as I see `invoices.bypass-acquisition-units` also [protect some procedure on mod-invoice](https://github.com/folio-org/mod-invoice/blob/master/src/main/java/org/folio/rest/impl/InvoiceHelper.java#L392)

Reply:
updating `orders.bypass-acquisition-units` and `invoices.bypass-acquisition-units` to more suitable type `procedural` and action `execute`